core/chains/evm/config/toml/defaults/Polygon_Zkevm_Cardona.toml

```diff
@@ -13,12 +13,13 @@ ContractConfirmations = 1
 ResendAfterThreshold = '3m'
 
 [GasEstimator]
+Mode = 'FeeHistory'
 PriceMin = '1 mwei'
```

Comment:
PriceMin is not enforceable by the new estimator and it will only be validated during attempt building. Unless the network does actually enforce a 1mwei min price, It would be better to set it to 0.